cyanlove/wp-security-bp
# Core >> Add i18n

### Changes:

`JSON` class
- `JSON` class does not do i18n anymore
- `pass` and `fail` methods from `JSON` class changed the arguments order
- `$plugin_name` removed from `JSON` class

`database`, `users` and `files` classes
- Refactor messages and `JSON` class calls

`wp-security-bp.pot`
- updated with current strings

New Files:

`wp-security-bp-es_ES.po`
`wp-security-bp-es_ES.mo`
- For demo purposes only

### Test

1. Set WordPress language to English
2. Pull the branch
3. Check messages from the plugin
4. Change WordPress language to Spanish
5. Check again messages from the plugin
